<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCoinsTargetWidget</class>
 <widget class="QFrame" name="SendCoinsTargetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="send_coins_group_to">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,1,0">
      <property name="margin">
       <number>14</number>
      </property>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Account name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <layout class="QHBoxLayout" name="pay_to_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QValidatedLineEdit" name="pay_to_address">
          <property name="toolTip">
           <string>The address to send the payment to  (e.g. PBZ8YVV3XT3WWWd2a1jo4N9WePiwKB3mJE)</string>
          </property>
          <property name="maxLength">
           <number>34</number>
          </property>
          <property name="placeholderText">
           <string>Enter a Pandacoin address (e.g. PBZ8YVV3XT3WWWd2a1jo4N9WePiwKB3mJE)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="address_book_button">
          <property name="toolTip">
           <string>Choose address from address book</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="paste_button">
          <property name="toolTip">
           <string>Paste address from clipboard</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+P</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="delete_button">
          <property name="toolTip">
           <string>Remove this recipient</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Account address</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="next_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="add_another_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add another</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>â–¾</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>New accounts will automatically be added to your address book</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="pay_to_name">
        <property name="placeholderText">
         <string>Enter an account name for this account address to add it to your address book</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="BitcoinAmountField" name="pay_amount"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
